sycl/source/detail/kernel_impl.hpp

```diff
@@ -31,6 +31,7 @@ class kernel_bundle_impl;
 
 using ContextImplPtr = std::shared_ptr<context_impl>;
 using ProgramImplPtr = std::shared_ptr<program_impl>;
+using ProgramPIPtr = sycl::detail::pi::PiProgram;
```

Comment:
Nitpick: can we do this instead? Seems a bit more straightforward.
```suggestion
using sycl::detail::pi::PiProgram;
```